Makefile

```diff
@@ -160,8 +160,8 @@ install: download-external-crds manifests kustomize ## Install CRDs into the K8s
 .PHONY: download-external-crds
 download-external-crds:
 	curl -s -L -o config/crd/external/subscriptions.eventing.kyma-project.io.crd.yaml https://raw.githubusercontent.com/kyma-project/kyma/main/installation/resources/crds/eventing/subscriptions.eventing.kyma-project.io.crd.yaml
-	curl -s -L -o config/crd/external/applications.applicationconnector.crd.yaml https://raw.githubusercontent.com/kyma-project/kyma/main/installation/resources/crds/application-connector/applications.applicationconnector.crd.yaml
-	curl -s -L -o config/crd/external/apirules.gateway.crd.yaml https://raw.githubusercontent.com/kyma-project/kyma/main/installation/resources/crds/api-gateway/apirules.gateway.crd.yaml
+	curl -s -L -o config/crd/for-tests/applications.applicationconnector.crd.yaml https://raw.githubusercontent.com/kyma-project/kyma/main/installation/resources/crds/application-connector/applications.applicationconnector.crd.yaml
+	curl -s -L -o config/crd/for-tests/apirules.gateway.crd.yaml https://raw.githubusercontent.com/kyma-project/kyma/main/installation/resources/crds/api-gateway/apirules.gateway.crd.yaml
```

Comment:
Why change it? The make target says `download-external-crds`!
Also can we already get external CRDs from their new repositories per module?

Reply:
They are only needed for tests, and we should use them in actual kustomize. I will update the urls in separate PR.